text/0000-cargo-sbom.md

```diff
@@ -0,0 +1,116 @@
+- Feature Name: `cargo-sbom`
+- Start Date: 2023-11-01
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC adds an option to Cargo that emits a Software Bill of Materials (SBOM) alongside compiled artifacts. Similar to how Cargo emits split debug info or "dep-info" (.d) files, this change emits an SBOM in a Cargo-specific format alongside outputs in the `target` directory. External tooling or Cargo subcommands can consume this Cargo SBOM file and transform it into other SBOM formats such as SPDX or CycloneDX.
```

Comment:
I strongly feel this RFC should _not_ say that Cargo will produce "an SBOM" but instead that Cargo will produce "an SBOM-precursor" or some similar language which makes clear the intent of the output is _not_ to be a redistributable SBOM itself, but instead to provide the necessary information to construct accurate SBOMs in an existing format for Rust programs built using Cargo.

There are discussions around the RFC below which specifically deal with this distinction, and I think starting off by clearly distinguishing the output here as an "SBOM-precursor" rather than an SBOM would help short-circuit future confusion.

Reply:
Thanks for this feedback. I've updated the summary to make it more clear that the output from this RFC is not intended as an sbom itself.